### ISSUE TITLE
Kebab/snake to camel/Pascal and back doesn't convert double capitals back correctly

### ISSUE THREAD

**Issue by allyourcodebaseisbelongtous** [finntenzor/change-case#10]
Using 2/19/2022, 04:02:46 on Mac
Kebab or snake to camel or Pascal and back doesn't convert double capitals back correctly

Example behavior:
```
open-a-review
OpenAReview
open-areview
```
```
open_a_review
OpenAReview
open_areview
```
```
open-a-review
openAReview
open-areview
```
```
open_a_review
openAReview
open_areview
```

**Comment by finntenzor** [comment]
This issue is a little difficult for me. I think this problem should only arise if there are single-letter words. I will try to fix the problem, but I may not be able to fix it in a short time.

**Comment by finntenzor** [comment]
Fixed in version 0.0.6.
